app/src/main/java/org/fossasia/openevent/app/contract/presenter/LoginPresenter.java

```diff
@@ -0,0 +1,15 @@
+package org.fossasia.openevent.app.contract.presenter;
+
+import org.fossasia.openevent.app.contract.view.LoginView;
+
+public interface LoginPresenter {
+
+    void start();
```

Comment:
@iamareebjamal Just to get clear - who will be calling these functions ??  View?